### ISSUE TITLE
shinyapp references problem

### ISSUE THREAD

**Issue by chantaldenhaan** [nealhaddaway/citationchaser#37]
Hello citationchaser team,

Last month I backward chased a set of 40 articles using the shinyapp, resulting in 2052 records being exported from lens.org. Yesterday I did the same with a set of 71 articles including the 40 from last month, and only 723 records were exported. I also got a weird message about 4212 references that make up 7059 unique IDs? However I cannot reproduce that last message today.

![image](https://github.com/user-attachments/assets/b33ce1fe-99f6-49d1-a3c1-9a63b940cc41)

It doesn't matter if I use PMIDs or DOI for article input, nor does changing browsers (I tried Chrome and Edge) make a difference. The number of results being exported is nowhere near the expected number. 

I hope that this is an issue that you can look into.

Many thanks, Chantal

**Comment by nealhaddaway** [comment]
Hi Chantal - thanks for flagging the issue. There's a known fault that was caused by a change in the API - we're working on fixing it now.
Thanks
________________________________
From: chantaldenhaan ***@***.***>
Sent: 24 July 2024 10:33
To: nealhaddaway/citationchaser ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nealhaddaway/citationchaser] shinyapp references problem (Issue #37)


Hello citationchaser team,

Last month I backward chased a set of 40 articles using the shinyapp, resulting in 2052 records being exported from lens.org. Yesterday I did the same with a set of 71 articles including the 40 from last month, and only 723 records were exported. I also got a weird message about 4212 references that make up 7059 unique IDs? However I cannot reproduce that last message today.

image.png (view on web)<https://github.com/user-attachments/assets/b33ce1fe-99f6-49d1-a3c1-9a63b940cc41>

It doesn't matter if I use PMIDs or DOI for article input, nor does changing browsers (I tried Chrome and Edge) make a difference. The number of results being exported is nowhere near the expected number.

I hope that this is an issue that you can look into.

Many thanks, Chantal

**Comment by lih34525** [comment]
Hi there,
Just wanted to follow-up as I seem to be getting issues with the references (backward chasing) tab of the shiny app as well:
"Your 72 articles contained a total of 1,348 references. Your RIS file is ready for download and contains 98 records exported from Lens.org."

Citations (forward chasing) is working well though!
Thanks,
Henry

**Comment by nealhaddaway** [comment]
Thanks Henry - the underlying Lens API changed and broken our code. Hoping to get it fixed later in the summer.
Thanks

https://www.linktr.ee/nealhaddaway
https://www.nealhaddaway.com

On 15 Aug 2024, at 16:58, lih34525 ***@***.***> wrote:

﻿

Hi there,
Just wanted to follow-up as I seem to be getting issues with the references (backward chasing) tab of the shiny app as well:
"Your 72 articles contained a total of 1,348 references. Your RIS file is ready for download and contains 98 records exported from Lens.org."

Citations (forward chasing) is working well though!
Thanks,
Henry

**Comment by christiangoelz** [comment]
Hi, any updates on this? Got similar confusing numbers today.

**Comment by nealhaddaway** [comment]
Afraid not - the backend is now fully functional, but I'm waiting for help rebuilding the front end. In the meantime you can perform this in Lens.org itself with a list of DOIs.
________________________________

**Comment by christiangoelz** [comment]
Perfect! Thanks for the quick answer. 
Is the R package already fixed?

**Comment by nealhaddaway** [comment]
No, sadly not. Again waiting until someone with free time can fix it (I'm freelance).
________________________________

**Comment by nealhaddaway** [comment]
Thanks - it's specifically Shiny we need help with now, sadly..
________________________________

**Comment by DrMattG** [comment]
Update coming very soon (today- we are testing this now and it should be pushed to the public version soon. Please let us know if you experience any further issues.

**Comment by lih34525** [comment]
Thanks for the update! It seems to be mostly working now, although for backwards citation searching it doesn't give me the total number of references, just the number of unique records after dedup. I assume there were 1,348 references since I tested this previously but it's just not showing now. Otherwise seems to be working!

"Your 72 input article(s) had NA references and 993 unique records after deduplication - they are ready for download"

**Comment by DrMattG** [comment]
> Thanks for the update! It seems to be mostly working now, although for backwards citation searching it doesn't give me the total number of references, just the number of unique records after dedup. I assume there were 1,348 references since I tested this previously but it's just not showing now. Otherwise seems to be working!
> 
> "Your 72 input article(s) had NA references and 993 unique records after deduplication - they are ready for download"

Thanks @lih34525 are you able to send the starting article IDs to us so we can try to recreate the problem?
